<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>V_Mainwindow</class>
 <widget class="QMainWindow" name="V_Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PluriNotes - FAYAX</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/windowicon.jpg</normaloff>../ressources/windowicon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New...</string>
     </property>
     <addaction name="actionArticle"/>
     <addaction name="actionMultimedia"/>
     <addaction name="actionTask"/>
     <addaction name="separator"/>
     <addaction name="actionRelation"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Asc_Desc_View"/>
    <addaction name="actionShow_Relations_View"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionArticle"/>
   <addaction name="actionMultimedia"/>
   <addaction name="actionTask"/>
   <addaction name="actionRelation"/>
   <addaction name="actionShow_Asc_Desc_View"/>
   <addaction name="actionShow_Relations_View"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the app</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionArticle">
   <property name="icon">
    <iconset>
     <normaloff>../ressources/newArticle.png</normaloff>../ressources/newArticle.png</iconset>
   </property>
   <property name="text">
    <string>Article</string>
   </property>
   <property name="toolTip">
    <string>Add a new Article</string>
   </property>
  </action>
  <action name="actionMultimedia">
   <property name="icon">
    <iconset>
     <normaloff>../ressources/newMultimedia.png</normaloff>../ressources/newMultimedia.png</iconset>
   </property>
   <property name="text">
    <string>Multimedia</string>
   </property>
   <property name="toolTip">
    <string>Add a new Multimedia</string>
   </property>
  </action>
  <action name="actionTask">
   <property name="icon">
    <iconset>
     <normaloff>../ressources/newTask.png</normaloff>../ressources/newTask.png</iconset>
   </property>
   <property name="text">
    <string>Task</string>
   </property>
   <property name="toolTip">
    <string>Add a new Task</string>
   </property>
  </action>
  <action name="actionShow_Asc_Desc_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../ressources/asc_desc.png</normaloff>../ressources/asc_desc.png</iconset>
   </property>
   <property name="text">
    <string>Show Asc/Desc View</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionShow_Relations_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../ressources/1_2.png</normaloff>../ressources/1_2.png</iconset>
   </property>
   <property name="text">
    <string>Show Relations View</string>
   </property>
   <property name="toolTip">
    <string>Show Relations View</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionRelation">
   <property name="icon">
    <iconset>
     <normaloff>../ressources/newRelation.png</normaloff>../ressources/newRelation.png</iconset>
   </property>
   <property name="text">
    <string>Relation</string>
   </property>
  </action>
  <action name="actionCouple">
   <property name="icon">
    <iconset>
     <normaloff>../ressources/newCouple.png</normaloff>../ressources/newCouple.png</iconset>
   </property>
   <property name="text">
    <string>Couple</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
